avm/res/container-service/managed-cluster/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -79,7 +79,7 @@ module testDeployment '../../../main.bicep' = [
           enableAutoScaling: true
           maxCount: 3
           maxPods: 50
-          minCount: 3
+          minCount: 1
```

Comment:
As I think the recommendation for the node count is at least `3` - I'd suggest adding a comment like
`minCount: 1 // Should be '3' but is reduced for testing` for this and the other changed lines. @ilhaan wha'ts your take?

Reply:
WAF PSFule needs this to be 3.